source/frontend/vite.config.ts

```diff
@@ -55,7 +55,6 @@ export default defineConfig({
         },
       },
     },
-    pool: 'vmThreads',
```

Comment:
Note, this seems to be the root cause of the console logs not being visible in the vite test terminal.